src/metadata.ts

```diff
@@ -153,6 +164,9 @@ export default async () => {
             AuthController: {
               signIn: { type: t['./auth/dto/sign-in-response'].SignInResponse },
               signUp: { type: t['./auth/dto/sign-up-response'].SignUpResponse },
+              generatePAT: {
+                type: t['./auth/dto/create-pat-response'].CreatePatResponse,
```

Comment:
hmm, where did `generatePAT` came from? You have `createPat` / `createPAT` everywhere

Reply:
this is the name of controller method

Reply:
ah I see - how about we unify the method name for creating the personal access token everywhere (in controller and service)?